### PR TITLE
My usual grammar pass, plus adding documentation for PRs without any

### PR DESCRIPTION
**Pull request by gwynne**

### CODE REVIEW

**Comment by VaporNationTravis** [comment]
### Proselint found issues

#### [week4-18-02-19.md](/twof/VaporNation/tree/patch-7/week4-18-02-19.md)

Line | Message | Severity |
| --- | ----- | ----- |
82 | More than 30 ppm of exclamations. Keep them under control. | warning


### Spell Checker found issues

#### [week4-18-02-19.md](/twof/VaporNation/tree/patch-7/week4-18-02-19.md)

Line | Typo |
| --- | ------ |
10 | rvices which controlled their **behavior** when requested from different | 
10 | g implementations were `class`**es;** `struct`-based services were | 
10 | ffered a manual control for a **behavior** already baked into Swift via | 
10 | . Now, to get the "singleton" **behavior,** use a `class` type to implem | 
10 | reviously default "instanced" **behavior** (a new instance of the servic | 
12 | **TL;DR:** The redundant singleton f | 
12 | services was removed, and the **behavior** when requesting a given servi | 
14 | [vapor/service#14](https://github.com/vapor/ser | 
32 | [vapor/fluent#380](https://github.com/vapor/flu | 
36 | ocol, significantly improving **debuggability** across the board, especially | 
38 | [vapor/core#92](https://github.com/vapor/cor | 
44 | [vapor/engine#237](https://github.com/vapor/eng | 
48 | ious HTTP status codes in the **3xx** series. Redirection can be co | 
48 | ts:)` method with the desired **maxium** redirection count (including | 
50 | [vapor/vapor#1480](https://github.com/vapor/vap | 
56 | - **Losslessly** optimizes DateMiddleware to o | 
56 | - Losslessly optimizes **DateMiddleware** to only calculate a new RFC11 | 
56 | eware to only calculate a new **RFC1123** timestamp once per second | 
56 | only calculate a new RFC1123 **timestamp** once per second | 
65 | In the above snippet, **DateMiddleware** won't be initialized until th | 
69 | [vapor/vapor#1507](https://github.com/vapor/vap | 
73 | Leaf Syntax Highlighting For **InteliJ** | 
77 | [vapor/leaf#76](https://github.com/vapor/lea | 
84 | [vapor/api-template#45](https://github.com/vapor/api | 
88 | [@SandorDobi](https://github.com/SandorDob | 
90 | [vapor/mysql#127](https://github.com/vapor/mys | 
96 | [vapor/auth#24](https://github.com/vapor/aut | 
100 | m/vapor/vapor/issues/1398), [@MaximBazarov](https://github.com/MaximBaza | 
107 | [vapor/vapor#1388](https://github.com/vapor/vap | 
125 | This PR ([vapor/fluent#385](https://github.com/vapor/flu | 
127 | [vapor/fluent#385](https://github.com/vapor/flu | 
135 | round it. Terse, created by [@John-Connolly](https://github.com/John-Conn | 
166 | for one reason or another. [@gperdomor](https://github.com/gperdomor | 
180 | [@**Cellane](https://github.com/Cellane)** | 
181 | [@bensyverson](https://github.com/bensyvers | 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: week4-18-02-19.md [hunk 1]
```diff
@@ -7,10 +7,16 @@
...
+
+TL;DR: The redundant singleton flag on services was removed, and the behavior when requesting a given service more than once now corresponds to Swift's standard value vs. reference semantics.
 
 [vapor/service#14](https://github.com/vapor/service/pull/14)
 
-### Both `struct` and `class` SQLite Models No Longer Require `init` Methods To Be Defined
+### SQLite Models No Longer Require `init` Methods
+
+This is true for both `struct`- and `class`-based models.
```
Comment (tanner0101) [inline week4-18-02-19.md]: Uh...what? How is that happening for classes.

File: week4-18-02-19.md [hunk 2]
```diff
@@ -26,69 +32,73 @@ final class Bar: SQLiteModel {
...
+
+`Debuggable` now conforms to Foundation's `LocalizedError` protocol, significantly improving debuggability across the board, especially including much improved log messages when an error is thrown from a unit test.
 
 [vapor/core#92](https://github.com/vapor/core/pull/92)
 
-<!---
-I don't understand what this one is doing. If you'd like to write a description and uncomment, that'd be awesome
-### Allow Session To Be Passed In
-#### HELP NEEDED: Write description
+### Custom `URLSession`s for `FoundationClient`
```
Comment (tanner0101) [inline week4-18-02-19.md]: This is Vapor 2

Comment (twof) [reply week4-18-02-19.md]: Gotchya. I'll scrap it then.

File: week4-18-02-19.md (hunk 1 above)
Comment (twof) [reply week4-18-02-19.md]: oh uhhhh. I completely misread the PR. What actually changed?
https://github.com/vapor/fluent/pull/380

**Pull request merged**